_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,252 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: atomic
+
+description: |
+  This rule checks that links that are embedded in a paragraph, list or cell have a way to distinguish them as links compared to the rest of the text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
+
+- has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role; and
+- is a [descendant][] of a `p`, `li` or `td` HTML elements (henceforth referred to as the _ancestor_ element) in the [flat tree][]; and
+- has [descendant][] [text nodes][text node] in the [flat tree][] that are [visible][]; and
+- the _ancestor_ element has other [descendant][] [visible][] [text nodes][text node] in the [flat tree][].
+
+## Expectation
+
+Each target element has either:
+
+- content (such as an image or text) visually located inside, or immediately before or after the test target, that identifies it as a link; or
+- a [distinguishing style][] not based on color alone, compared to the _ancestor_ element and to any of its [descendant][] HTML elements that have [visible][] [text nodes][text node]; or
```

Comment:
> the ancestor element

Which ancestor element? The one from the applicability? What if there is more than one?

Also, the way this is worded, if the paragraph has two links with the same style, this would not be true and those things could fail the rule.

Reply:
Nice catch (both)